endpoints.tf

```diff
@@ -13,3 +13,147 @@ resource "aws_vpc_endpoint" "transfer_server" {
   vpc_endpoint_type   = "Interface"
   vpc_id              = aws_vpc.default.id
 }
+
+
+data "aws_region" "current" {}
```

Comment:
This should be moved to main.tf as it's a shared resource, not specific to VPC endpoints.